src/Microsoft.Azure.WebJobs.Extensions.Storage/Queues/Listeners/QueueListener.cs

```diff
@@ -40,6 +40,7 @@ internal sealed partial class QueueListener : IListener, ITaskSeriesCommand, INo
         private readonly FunctionDescriptor _functionDescriptor;
         private readonly string _functionId;
         private readonly ScaleMonitorDescriptor _scaleMonitorDescriptor;
+        private readonly CancellationTokenSource _shutdownCancellationTokenSounce;
```

Comment:
typo: _shutdownCancellationTokenSou**r**ce (source not sounce)

Reply:
Thanks! It's fixed now.